mlflow/h2o.py

```diff
@@ -60,6 +69,21 @@ def log_model(h2o_model, artifact_path, **kwargs):
     :param h2o_model: H2O model to be saved.
     :param artifact_path: Run-relative artifact path.
     :param kwargs: kwargs to pass to ``h2o.save_model`` method.
+
+    >>> import mlflow
```

Comment:
If we remove the mlflow.log_param calls below we can remove the import mlflow statement.

Reply:
Agreed. Will do